data/containers/app.py

```diff
@@ -0,0 +1,28 @@
+from flask import Flask, render_template
+import os, sys, ssl
+
+app = Flask(__name__)
+message = ""
+
+@app.route("/")
+def index():
+    return render_template("index.html", message = message)
+
+if __name__ == "__main__":
+    ctx = ssl.create_default_context()
```

Comment:
I don't see where the context `ctx` is actually used. Is this still required?

Reply:
`ssl.create_default_context()` will create an instance with secure default settings which are overrode below, to avoid ssl warnings etc..

Reply:
ready for review